filesystem/ios/Plugin/FilesystemPlugin.m

```diff
@@ -18,4 +18,5 @@
            CAP_PLUGIN_METHOD(checkPermissions, CAPPluginReturnPromise);
            CAP_PLUGIN_METHOD(requestPermissions, CAPPluginReturnPromise);
            CAP_PLUGIN_METHOD(downloadFile, CAPPluginReturnPromise);
+           CAP_PLUGIN_METHOD(removeAllListeners, CAPPluginReturnNone);
```

Comment:
```suggestion
```
Remove this as we are removing them on main branch as it won't be needed for Capacitor 6
https://github.com/ionic-team/capacitor-plugins/pull/1867